src/elvis_style.erl

```diff
@@ -982,3 +1003,24 @@ filter_repeated(NodesLocs) ->
 is_children(Parent, Node) ->
     Zipper = elvis_code:code_zipper(Parent),
     [] =/= zipper:filter(fun(Child) -> Child == Node end, Zipper).
+
+%% No nested try...catch blocks
+
+check_nested_try_catchs(ResultFun, TryExp) ->
+    Predicate = fun(Node) -> ktn_code:type(Node) == 'try' end,
+    FilterMapFun =
+        fun (Node) ->
+                case Node /= TryExp of
```

Comment:
You can avoid using this `case` expression here by using pattern matching in the `fun`'s clauses.